src/main/java/ca/bc/gov/backendstartapi/config/SecurityConfig.java

```diff
@@ -24,7 +24,7 @@
 /** This class contains all configurations related to security and authentication. */
 @Configuration
 @EnableWebSecurity
-@EnableMethodSecurity(prePostEnabled = true)
+@EnableMethodSecurity
```

Comment:
Why we need this?

Reply:
Thing is we didn't need `prePostEnabled=true`, as it's the default option. I don't remember how I got into this class during the development (I believe I moved it around at some point), but the IDE alerted me to it and how it was unnecessary.

That said, it isn't wrong. I believe it's valid to keep the default assignment explicit if we deem it advantageous for making the configuration clearer, for example.